projects/farmworker-wallet-os.md

```diff
@@ -0,0 +1,120 @@
+
+# Release Methodology
+_If the project already exists, please provide details on the release methodology and mechanics._
+
+# Initial Maintainers
+* [Ockert van Schalkwyk](https://github.com/skullquake)
+* [Jorge Flores](https://github.com/jorgefl0)
+
+# Proposed Project Governance
+The leadership team is composed of the individuals responsible for the development of the Mendix digital wallet used in the Preparese solution. The team has a demonstrated track record of delivering enterprise level digital trust solutions to the market. The proposed members are from Entidad and the Mendix developer community. By drawing from these two groups, we hope to introduce a balanced and diverse perspective that is representative of the three core communities this project hopes to serve.   
+
+ 
+All are committed to the Farmworker Wallet OS project and each leader plays a different role in its success: 
+
+* As the principal, Jorge Flores is responsible for maintaining the vision of the project, sourcing outside advisors, participants, and new ideas for collaboration and industries to serve. His extensive background as a CTO and his passion for digital trust technologies, communication platforms, and mobile application frameworks makes him a prime candidate to be the technical advisor for this project.  
+
+* As the Developer Lead, Ockert van Schalkwyk is responsible for maintaining an Enterprise level standard for the project’s software code.  He will review and approve changes, forks, and merges.  His background as senior Mendix developer with a focus on systems integrations, makes him the ideal candidate for the role.    
+
+* As a Mendix Solution architect, Stephan Bruijnis is responsible for guiding architectural integrity of the project.  He will ensure the project upholds and maintains targeted privacy, security, and interoperability standards. His extensive experience as a Senior Mendix architect makes him ideal for this project.    
+
+* As the Methodology and Mechanics Lead, Ned Gosaynie is responsible for maintaining documentation and versioning for the project.  He will review and approve the technical documentation needed to support the project needs. Additionally, he will manage versioning of the software. His experience as the of Head of Platform Services makes him the ideal candidate for this position.   
+
+ 
+The team’s philosophy for the Farmworker Wallet OS project will be a continuation of the social good principles that have driven the project since its origin. Having worked together for years, the team has a well-established working relationship. Decisions will be evaluated through the lens of farmworker benefit, security, privacy, and impact.   
+
+# Links to Documented Governance Practices
+_Include links to any documented governance practices._
+
+# Preferred Maturity Level
+Labs
+
+# Communication Channels
+Slack
```

Comment:
Will you migrate to the OWF Discord server for communication regarding the project?

Reply:
Yes, that is fine. Most of the Mendix Community is on Slack, but it would be acceptable to us to move communications to Discord. It makes sense given that Discord is also used by DIF, Hyperledger and OWF.